cli/functionary/package.py

```diff
@@ -72,41 +72,16 @@ def publish(ctx, path):
     Use the -t option to specify a token or set the FUNCTIONARY_TOKEN
     environment variable after logging in to Functionary.
     """
-    token = get_config_value("token")
     host = get_config_value("host")
 
     full_path = pathlib.Path(path).resolve()
     tarfile_name = full_path.joinpath(f"{full_path.name}.tar.gz")
     with tarfile.open(str(tarfile_name), "w:gz") as tar:
         tar.add(str(full_path), arcname="")
 
+    upload_file = open(tarfile_name, "rb")
```

Comment:
This looks like it was true before, but this file gets opened and then isn't closed anywhere.  Seems like something to address.  You can use `open` in a [context manager](https://book.pythontips.com/en/latest/context_managers.html) to fix this.